docs/architecture.mdx

```diff
@@ -0,0 +1,159 @@
+
+The shell is responsible for the fundamental functions of a distributed software application, including:
+
+- Peer-to-peer communication that lets nodes exchange information
+- Storage functionality that lets nodes store blocks, operations, and the current state of the chain
+- A synchronization heuristic that starts nodes and keeps them in sync with the network
+- A validator that checks that blocks are valid with help from the rules in the economic protocol
+
+In particular, the validator is responsible for resolving the available blocks into a single linear sequence of blocks.
+It chooses between the various blocks that baking nodes create, uses the protocol to verify and score them, and selects the tree head with the highest score.
+Then it uses that linear chain in all of its work with the protocol, so the protocol is never aware of multiple branches.
+
+## Tezos clients and servers
+
+In addition to the functions of the protocol and shell, a Tezos node also acts as a server to respond to queries and requests from clients.
+A client can query the chain’s state and can inject blocks and operations into a node.
+
+Tezos uses this client-server architecture for these main reasons:
+
+- It improves security by separating the node, which is exposed to the internet, from the baker, which has access to the client keys.
+The node and the baker can be on different computers, which allows the node to manage communication and shield bakers from network attacks.
+- It allows bakers to have different implementations.
+For example, different bakers may implement different transaction selection strategies.
+- It allows clients and other tools to interact with the node and inspect its state.
+
+The node accepts calls from clients through its RPC interface.
+It has control over which clients to accept calls from, which calls to accept, or whether to accept RPC calls at all.
+For more information on the RPC interface, see [The RPC protocol](./architecture/rpc).
+
+### The baker daemon
+
+The baker daemon is an Octez program that is responsible for creating and proposing new blocks based on the operations proposed by different clients.
+The baker daemon is associated with an account, which means that it has access to the account’s private key, which it uses to sign blocks and operations.
+
+### The accuser daemon
+
+The accuser daemon is an Octez program that monitors new blocks and looks for problems, such as when bakers try to add more than one block at a time.
+When it finds a problem, it submits a denunciation to other nodes to refuse the new blocks and punish the offending node.
+
+### The Octez client
+
+The Octez client is a command-line tool that developers can use for many Tezos-related tasks, including:
+
+- Deploying, calling, testing, and interacting with contracts
+- Deploying and interacting with Smart Rollups
+- Working with accounts
+- Getting contract balances and data
+- Calling RPC endpoints directly
+- Running sapling transactions
```

Comment:
FYI, in the Octez doc, we always spell Sapling with a capital, but up to you to choose your convention (be consistent, though).

Reply:
I see that opentezos capitalizes Sapling as well -- I'll make that update.